apps/agentic-chat/src/lib/wagmi-config.ts

```diff
@@ -0,0 +1,10 @@
+import { http, createConfig } from 'wagmi';
+import { arbitrum, mainnet } from 'wagmi/chains';
+
+export const wagmiConfig = createConfig({
+  chains: [mainnet, arbitrum],
+  transports: {
+    [mainnet.id]: http(),
+    [arbitrum.id]: http(),
```

Comment:
Basically this PR works on arbitrum only for devving purposes, to be improved in https://github.com/shapeshift/agentic-chat/issues/11